src/EventHub.Application/EventHubApplicationAutoMapperProfile.cs

```diff
@@ -27,10 +27,20 @@ public EventHubApplicationAutoMapperProfile()
                 .Ignore(x => x.OrganizationDisplayName)
                 .Ignore(x => x.IsLiveNow);
             CreateMap<Event, EventDetailDto>()
+                .ForMember(x => x.Tracks, memberOptions => memberOptions.MapFrom(m => m.Tracks))
                 .Ignore(x => x.OrganizationId)
                 .Ignore(x => x.OrganizationName)
                 .Ignore(x => x.OrganizationDisplayName);
 
+            CreateMap<Track, TrackDto>()
+                .ForMember(x => x.Sessions, memberOptions => memberOptions.MapFrom(m => m.Sessions));
```

Comment:
Property names are the same (`Sessions`) in both classes, is it needed to specify?